config/deploy/prod.rb

```diff
@@ -2,8 +2,8 @@
 
 set :default_env, 'APP_ENV' => ENV['APP_ENV']
 
-server 'sul-sdr-services.stanford.edu', user: 'sdr2service', roles: %w{app}
+server 'sdr-services-app-prod.stanford.edu', user: 'sdr2service', roles: %w{app}
 
 Capistrano::OneTimeKey.generate_one_time_key!
 
-set :honeybadger_env, 'prod'
```

Comment:
I'm pretty sure we still need `honeybadger_env`.

Reply:
whoops - will fix,  thanks.

Reply:
@jcoyne fixed - thanks for spotting that.